polkadot-parachain/src/chain_spec/default.rs

```diff
@@ -0,0 +1,122 @@
+// Copyright 2019-2021 Parity Technologies (UK) Ltd.
+// This file is part of Cumulus.
+
+// Cumulus is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Cumulus is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Cumulus.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Default parachain ChainSpec based on testing rococo parachain runtime
+
+use crate::chain_spec::{get_from_seed, Extensions, SAFE_XCM_VERSION};
+use cumulus_primitives_core::ParaId;
+use hex_literal::hex;
+use parachains_common::AccountId;
+use polkadot_service::chain_spec::get_account_id_from_seed;
+use rococo_parachain_runtime::AuraId;
+use sc_chain_spec::ChainType;
+use sp_core::{crypto::UncheckedInto, sr25519};
+
+/// Specialized `ChainSpec` for the normal parachain runtime.
+pub type DefaultChainSpec =
+	sc_service::GenericChainSpec<rococo_parachain_runtime::GenesisConfig, Extensions>;
+
+pub fn get_chain_spec() -> DefaultChainSpec {
+	DefaultChainSpec::from_genesis(
+		"Local Testnet",
+		"local_testnet",
+		ChainType::Local,
+		move || {
+			testnet_genesis(
+				get_account_id_from_seed::<sr25519::Public>("Alice"),
+				vec![get_from_seed::<AuraId>("Alice"), get_from_seed::<AuraId>("Bob")],
+				vec![
+					get_account_id_from_seed::<sr25519::Public>("Alice"),
+					get_account_id_from_seed::<sr25519::Public>("Bob"),
+					get_account_id_from_seed::<sr25519::Public>("Charlie"),
+					get_account_id_from_seed::<sr25519::Public>("Dave"),
+					get_account_id_from_seed::<sr25519::Public>("Eve"),
+					get_account_id_from_seed::<sr25519::Public>("Ferdie"),
+					get_account_id_from_seed::<sr25519::Public>("Alice//stash"),
+					get_account_id_from_seed::<sr25519::Public>("Bob//stash"),
+					get_account_id_from_seed::<sr25519::Public>("Charlie//stash"),
+					get_account_id_from_seed::<sr25519::Public>("Dave//stash"),
+					get_account_id_from_seed::<sr25519::Public>("Eve//stash"),
+					get_account_id_from_seed::<sr25519::Public>("Ferdie//stash"),
+				],
+				1000.into(),
+			)
+		},
+		Vec::new(),
+		None,
+		None,
+		None,
+		None,
+		Extensions { relay_chain: "westend".into(), para_id: 1000 },
+	)
+}
+
+pub fn staging_test_net() -> DefaultChainSpec {
```

Comment:
this fn and the one above, seems like doing similar things, but the names are so different.
would be great to have some docs

Reply:
I dont now how is this cfg used, but I renamed it accordingly `staging_rococo_parachain_local`